packages/board-server/package.json

```diff
@@ -58,6 +58,7 @@
       "files": [
         "scripts/build.ts",
```

Comment:
Seems like this could be a glob instead?

```json
"files": [
 "scripts/**/*.ts",
 "src/**/*.ts",
 "tsconfig.json",
 "package.json"
]
```

Reply:
Is there any reason we don't just do `**/*.ts` here to pick up everything?

Reply:
For example, the `vite:build` target has `src/app/**/*.ts"`. That would cause these two targets to overlap.

Why does this separation exist?

Reply:
> Is there any reason we don't just do `**/*.ts` here to pick up everything?

Good question. Because that would also match e.g. `dist/foo.d.ts`, an output file.

Reply:
> Why does this separation exist?

I'm not sure. I'm guessing it might be that vite is handling the frontend code, and `build.ts` is handling the backend code?

BTW I think we should split these packages -- seems like the board server should be independent of the generated app view.

Reply:
But even with your change it would pick up `"src/app/**/*.ts"`, which is currently part of the fileset for `build:vite`. Do we want that?

Reply:
Discussed off-thread. The distinction is that the board-server package contains both frontend and backend code, and there are different build targets for each. We need these separate targets because the code is intermixed. We should probably fix this by reorganizing.

For now, just add the new files explicitly, and revisit this when we reorganize.